README.md

```diff
@@ -1,5 +1,58 @@
+![GitHub forks](https://img.shields.io/github/forks/beyond-io/picATrip?style=social) ![GitHub issues](https://img.shields.io/github/issues-raw/beyond-io/picATrip?logo=GitHub) ![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/beyond-io/picATrip?logo=GitHub) ![GitHub top language](https://img.shields.io/github/languages/top/beyond-io/picATrip)
 # picATrip
-picATrip - Beyond 05, team 2 project
+![logo- picATrip](https://user-images.githubusercontent.com/48455448/110465508-26ae0a80-80dd-11eb-8268-586b26a1e59e.png)
 
-A platform for sharing experiences from trips in a form of photos based on various parameters as locations/duration time/difficulty, and provides a trip search functionality and commenting functionality. 
-It allows users to have the oppurtunity to contribute their trip experience and thoughts as well as learn about trips of others across the country, and adding comment if desired.
+### **Who are we?**
+
+   **picATrip** is a web platform for **sharing experiences from _trips_** in a form of **photos** based
+   on various parameters as:
+* Location
+* Duration time
+* Difficulty
+
+ #### Other featurs It provides:
```

Comment:
features it

Reply:
can you elaborate on what do you mean by that? we discussed as a team several general features that I mentioned but we didn't get the chance to describe each one of them in detail assuming it can be changed in the future whether we're talking about how big of a feature we want or whether we would think along the way on more doable features, so what exactly missing - a more clear separation to features category or actually details about each feature, because as I said we're a bit lacking in this.

Reply:
I think he meant that there is a typo, "features" and not "featurs".

Reply:
I actually used a grammar checker after some typos at the first fixed versions, but apparently, the '#' sign was disrupting it, thanks